client/my-sites/plan-features/actions.jsx

```diff
@@ -72,10 +64,25 @@ const PlanFeaturesActionsButton = ( {
 		onUpgradeClick();
 	};
 
+	if ( isMarketplaceFlow( flowName ) ) {
+		if ( current ) {
+			return (
+				<Button className={ classes } disabled>
+					{ translate( 'Your current plan' ) }
```

Comment:
What does this have to do with mobile views? 

Can we deal with this problem after we get the buttons working?